Issue while deploying on fly io

Path: /documentation/guides/self-hosting/flyio

not able to deploy this on fly io

2023-08-15T15:44:27.353 app[148ed6e1f7d248] mia [info] You may have to run ${S("prisma generate")} for your changes to take effect.`,this.config.clientVersion);return r}parseEngineResponse(r){if(!r)throw new oe("Response from the Engine was empty",{clientVersion:this.config.clientVersion});try{return JSON.parse(r)}catch{throw new oe("Unable to JSON.parse response from engine",{clientVersion:this.config.clientVersion})}}convertDatasources(r){let n=Object.create(null);for(let{name:i,url:o}of r)n[i]=o;return n}async loadEngine(){if(!this.engine){this.QueryEngineConstructor||(this.library=await this.libraryLoader.loadLibrary(),this.QueryEngineConstructor=this.library.QueryEngine);try{let r=new WeakRef(this);this.engine=new this.QueryEngineConstructor({datamodel:this.datamodel,env:process.env,logQueries:this.config.logQueries??!1,ignoreEnvVarErrors:!0,datasourceOverrides:this.datasourceOverrides,logLevel:this.logLevel,configDir:this.config.cwd,engineProtocol:this.engineProtocol},n=>{r.deref()?.logger(n)}),ca++}catch(r){let n=r,i=this.parseInitError(n.message);throw typeof i=="string"?n:new Q(i.message,this.config.clientVersion,i.error_code)}}}logger(r){let n=this.parseEngineResponse(r);if(!!n){if("span"in n){this.config.tracingHelper.createEngineSpan(n);return}n.level=n?.level.toLowerCase()??"unknown",Bp(n)?this.logEmitter.emit("query",{timestamp:new Date,query:n.query,params:n.params,duration:Number(n.duration_ms),target:n.module_path}):Vp(n)?this.loggerRustPanic=new be(this.getErrorMessageWithLink(`${n.message}: ${n.reason} in ${n.file}:${n.line}:${n.column}`),this.config.clientVersion):this.logEmitter.emit(n.level,{timestamp:new Date,message:n.message,target:n.module_path})}}getErrorMessageWithLink(r){return sa({platform:this.platform,title:r,version:this.config.clientVersion,engineVersion:this.versionInfo?.commit,database:this.config.activeProvider,query:this.lastQuery})}parseInitError(r){try{return JSON.parse(r)}catch{}return r}parseRequestError(r){try{return JSON.parse(r)}catch{}return r}on(r,n){r==="beforeExit"?this.beforeExitListener=n:this.logEmitter.on(r,n)}async start(){if(await this.libraryInstantiationPromise,await this.libraryStoppingPromise,this.libraryStartingPromise)return qe(`library already starting, this.libraryStarted: ${this.libraryStarted}`),this.libraryStartingPromise;if(this.libraryStarted)return;let r=async()=>{qe("library starting");try{let n={traceparent:this.config.tracingHelper.getTraceParent()};await this.engine?.connect(JSON.stringify(n)),this.libraryStarted=!0,qe("library started")}catch(n){let i=this.parseInitError(n.message);throw typeof i=="string"?n:new Q(i.message,this.config.clientVersion,i.error_code)}finally{this.libraryStartingPromise=void 0}};return this.libraryStartingPromise=this.config.tracingHelper.runInChildSpan("connect",r),this.libraryStartingPromise}async stop(){if(await this.libraryStartingPromise,await this.executingQueryPromise,this.libraryStoppingPromise)return qe("library is already stopping"),this.libraryStoppingPromise;if(!this.libraryStarted)return;let r=async()=>{await new Promise(i=>setTimeout(i,5)),qe("library stopping");let n={traceparent:this.config.tracingHelper.getTraceParent()};await this.engine?.disconnect(JSON.stringify(n)),this.libraryStarted=!1,this.libraryStoppingPromise=void 0,qe("library stopped")};return this.libraryStoppingPromise=this.config.tracingHelper.runInChildSpan("disconnect",r),this.libraryStoppingPromise}async getDmmf(){await this.start();let r=this.config.tracingHelper.getTraceParent(),n=await this.engine.dmmf(JSON.stringify({traceparent:r}));return this.config.tracingHelper.runInChildSpan({name:"parseDmmf",internal:!0},()=>JSON.parse(n))}version(){return this.versionInfo=this.library?.version(),this.versionInfo?.version??"unknown"}debugPanic(r){return this.library?.debugPanic(r)}async request(r,{traceparent:n,interactiveTransaction:i}){qe(`sending request, this.libraryStarted: ${this.libraryStarted}`);let o=JSON.stringify({traceparent:n}),s=JSON.stringify(r);try{await this.start(),this.executingQueryPromise=this.engine?.query(s,o,i?.id),this.lastQuery=s;let a=this.parseEngineResponse(await this.executingQueryPromise);if(a.errors)throw a.errors.length===1?this.buildQueryError(a.errors[0]):new oe(JSON.stringify(a.errors),{clientVersion:this.config.clientVersion});if(this.loggerRustPanic)throw this.loggerRustPanic;return{data:a,elapsed:0}}catch(a){if(a instanceof Q)throw a;if(a.code==="GenericFailure"&&a.message?.startsWith("PANIC:"))throw new be(this.getErrorMessageWithLink(a.message),this.config.clientVersion);let l=this.parseRequestError(a.message);throw typeof l=="string"?a:new oe(`${l.message}

2023-08-15T15:44:27.353 app[148ed6e1f7d248] mia [info] ^

2023-08-15T15:44:27.353 app[148ed6e1f7d248] mia [info] PrismaClientInitializationError: Error opening a TLS connection: unexpected EOF

2023-08-15T15:44:27.353 app[148ed6e1f7d248] mia [info] at r (/triggerdotdev/node_modules/.pnpm/@prisma+client@4.16.2_prisma@4.16.0/node_modules/@prisma/client/runtime/library.js:108:2574)

2023-08-15T15:44:27.400 app[148ed6e1f7d248] mia [info]  ELIFECYCLE  Command failed with exit code 1.

2023-08-15T15:44:27.718 app[148ed6e1f7d248] mia [info] INFO Main child exited normally with code: 1

2023-08-15T15:44:27.719 app[148ed6e1f7d248] mia [info] INFO Starting clean up.

2023-08-15T15:44:27.720 app[148ed6e1f7d248] mia [info] WARN hallpass exited, pid: 256, status: signal: 15 (SIGTERM)

2023-08-15T15:44:27.727 app[148ed6e1f7d248] mia [info] 2023/08/15 15:44:27 listening on [fdaa:2:4fd7:a7b:184:f397:2946:2]:22 (DNS: [fdaa::3]:53)

2023-08-15T15:44:28.720 app[148ed6e1f7d248] mia [info] [ 12.308936] reboot: Restarting system

2023-08-15T15:44:28.928 runner[148ed6e1f7d248] mia [info] machine did not have a restart policy, defaulting to restart

2023-08-15T15:44:36.497 app[148ed6e1f7d248] mia [info] [ 0.039871] PCI: Fatal: No config space access function found

2023-08-15T15:44:36.718 app[148ed6e1f7d248] mia [info] INFO Starting init (commit: b437b5b)...

2023-08-15T15:44:36.737 app[148ed6e1f7d248] mia [info] INFO Preparing to run: `docker-entrypoint.sh ./scripts/entrypoint.sh` as node

2023-08-15T15:44:36.745 app[148ed6e1f7d248] mia [info] INFO [fly api proxy] listening at /.fly/api

2023-08-15T15:44:36.752 app[148ed6e1f7d248] mia [info] 2023/08/15 15:44:36 listening on [fdaa:2:4fd7:a7b:184:f397:2946:2]:22 (DNS: [fdaa::3]:53)

2023-08-15T15:44:36.755 app[148ed6e1f7d248] mia [info] + [ -n ]

2023-08-15T15:44:36.755 app[148ed6e1f7d248] mia [info] + pnpm --filter @trigger.dev/database db:migrate:deploy

2023-08-15T15:44:38.170 app[148ed6e1f7d248] mia [info] > @trigger.dev/database@0.0.0 db:migrate:deploy /triggerdotdev/packages/database

2023-08-15T15:44:38.170 app[148ed6e1f7d248] mia [info] > prisma migrate deploy

2023-08-15T15:44:38.808 app[148ed6e1f7d248] mia [info] Prisma schema loaded from prisma/schema.prisma

2023-08-15T15:44:38.842 app[148ed6e1f7d248] mia [info] Datasource "db": PostgreSQL database "postgres", schema "public" at "wallofreviews-trigger-v2-fly-demo-db.flycast:5432"

2023-08-15T15:44:39.088 app[148ed6e1f7d248] mia [info] Error: P1011: Error opening a TLS connection: unexpected EOF

2023-08-15T15:44:39.107 app[148ed6e1f7d248] mia [info] undefined

2023-08-15T15:44:39.109 app[148ed6e1f7d248] mia [info] /triggerdotdev/packages/database:

2023-08-15T15:44:39.109 app[148ed6e1f7d248] mia [info]  ERR_PNPM_RECURSIVE_RUN_FIRST_FAIL  @trigger.dev/database@0.0.0 db:migrate:deploy: `prisma migrate deploy`

2023-08-15T15:44:39.109 app[148ed6e1f7d248] mia [info] Exit status 1

2023-08-15T15:44:39.121 app[148ed6e1f7d248] mia [info] + cp packages/database/prisma/schema.prisma apps/webapp/prisma/

2023-08-15T15:44:39.126 app[148ed6e1f7d248] mia [info] + cp node_modules/@prisma/engines/libquery_engine-debian-openssl-1.1.x.so.node apps/webapp/prisma/

2023-08-15T15:44:39.143 app[148ed6e1f7d248] mia [info] + pnpm --filter webapp db:seed

2023-08-15T15:44:39.656 app[148ed6e1f7d248] mia [info] > webapp@1.0.0 db:seed /triggerdotdev/apps/webapp

2023-08-15T15:44:39.656 app[148ed6e1f7d248] mia [info] > node prisma/seed.js

2023-08-15T15:44:39.734 app[148ed6e1f7d248] mia [info] 🔌 setting up prisma client to postgres://postgres@wallofreviews-trigger-v2-fly-demo-db.flycast:5432

2023-08-15T15:44:39.740 app[148ed6e1f7d248] mia [info] 🔌 prisma client connected

2023-08-15T15:44:39.749 app[148ed6e1f7d248] mia [info] prisma:info Starting a postgresql pool with 3 connections.

2023-08-15T15:44:39.933 app[148ed6e1f7d248] mia [info] prisma:error

2023-08-15T15:44:39.933 app[148ed6e1f7d248] mia [info] Invalid `prisma.integrationDefinition.upsert()` invocation:

2023-08-15T15:44:39.933 app[148ed6e1f7d248] mia [info] Error opening a TLS connection: unexpected EOF

2023-08-15T15:44:39.936 app[148ed6e1f7d248] mia [info] PrismaClientInitializationError:

2023-08-15T15:44:39.936 app[148ed6e1f7d248] mia [info] Invalid `prisma.integrationDefinition.upsert()` invocation:

2023-08-15T15:44:39.936 app[148ed6e1f7d248] mia [info] Error opening a TLS connection: unexpected EOF

2023-08-15T15:44:39.936 app[148ed6e1f7d248] mia [info] at Rn.handleRequestError (/triggerdotdev/apps/webapp/prisma/seed.js:174:7589)

2023-08-15T15:44:39.936 app[148ed6e1f7d248] mia [info] at Rn.handleAndLogRequestError (/triggerdotdev/apps/webapp/prisma/seed.js:174:6742)

2023-08-15T15:44:39.936 app[148ed6e1f7d248] mia [info] at /triggerdotdev/apps/webapp/prisma/seed.js:177:3006

2023-08-15T15:44:39.936 app[148ed6e1f7d248] mia [info] at async /triggerdotdev/apps/webapp/prisma/seed.js:177:3221

2023-08-15T15:44:39.936 app[148ed6e1f7d248] mia [info] at async t._executeRequest (/triggerdotdev/apps/webapp/prisma/seed.js:177:11390)

2023-08-15T15:44:39.936 app[148ed6e1f7d248] mia [info] at async fg (/triggerdotdev/apps/webapp/prisma/seed.js:519:60716)

2023-08-15T15:44:39.936 app[148ed6e1f7d248] mia [info] at async pg (/triggerdotdev/apps/webapp/prisma/seed.js:519:61621) {

2023-08-15T15:44:39.936 app[148ed6e1f7d248] mia [info] clientVersion: '4.16.0',

2023-08-15T15:44:39.936 app[148ed6e1f7d248] mia [info] errorCode: undefined

2023-08-15T15:44:39.936 app[148ed6e1f7d248] mia [info] }

2023-08-15T15:44:39.943 app[148ed6e1f7d248] mia [info] undefined

2023-08-15T15:44:39.944 app[148ed6e1f7d248] mia [info] /triggerdotdev/apps/webapp:

2023-08-15T15:44:39.944 app[148ed6e1f7d248] mia [info]  ERR_PNPM_RECURSIVE_RUN_FIRST_FAIL  webapp@1.0.0 db:seed: `node prisma/seed.js`

2023-08-15T15:44:39.944 app[148ed6e1f7d248] mia [info] Exit status 1

2023-08-15T15:44:39.970 app[148ed6e1f7d248] mia [info] + cd /triggerdotdev/apps/webapp

2023-08-15T15:44:39.970 app[148ed6e1f7d248] mia [info] + exec dumb-init pnpm run start

2023-08-15T15:44:40.380 app[148ed6e1f7d248] mia [info] > webapp@1.0.0 start /triggerdotdev/apps/webapp

2023-08-15T15:44:40.380 app[148ed6e1f7d248] mia [info] > cross-env NODE_ENV=production node --max-old-space-size=8192 ./build/server.js

2023-08-15T15:44:41.676 app[148ed6e1f7d248] mia [info] 🔌 setting up prisma client to postgres://postgres@wallofreviews-trigger-v2-fly-demo-db.flycast:5432

2023-08-15T15:44:41.680 app[148ed6e1f7d248] mia [info] 🔌 prisma client connected

2023-08-15T15:44:41.925 app[148ed6e1f7d248] mia [info] {"runnerOptions":{"connectionString":"postgres://postgres:SGbxbMIWU8vqBjh@wallofreviews-trigger-v2-fly-demo-db.flycast:5432","concurrency":5,"pollInterval":1000,"noPreparedStatements":false},"timestamp":"2023-08-15T15:44:41.925Z","name":"webapp","message":"Initializing worker queue with options"}

2023-08-15T15:44:42.389 app[148ed6e1f7d248] mia [info] No PostHog API key, so analytics won't track

2023-08-15T15:44:44.301 app[148ed6e1f7d248] mia [info] ✅ app ready: http://localhost:3000

2023-08-15T15:44:45.939 app[148ed6e1f7d248] mia [info] prisma:info Starting a postgresql pool with 3 connections.

2023-08-15T15:44:46.355 app[148ed6e1f7d248] mia [info] /triggerdotdev/node_modules/.pnpm/@prisma+client@4.16.2_prisma@4.16.0/node_modules/@prisma/client/runtime/library.js:108

2023-08-15T15:44:46.355 app[148ed6e1f7d248] mia [info] You may have to run ${S("prisma generate")} for your changes to take effect.`,this.config.clientVersion);return r}parseEngineResponse(r){if(!r)throw new oe("Response from the Engine was empty",{clientVersion:this.config.clientVersion});try{return JSON.parse(r)}catch{throw new oe("Unable to JSON.parse response from engine",{clientVersion:this.config.clientVersion})}}convertDatasources(r){let n=Object.create(null);for(let{name:i,url:o}of r)n[i]=o;return n}async loadEngine(){if(!this.engine){this.QueryEngineConstructor||(this.library=await this.libraryLoader.loadLibrary(),this.QueryEngineConstructor=this.library.QueryEngine);try{let r=new WeakRef(this);this.engine=new this.QueryEngineConstructor({datamodel:this.datamodel,env:process.env,logQueries:this.config.logQueries??!1,ignoreEnvVarErrors:!0,datasourceOverrides:this.datasourceOverrides,logLevel:this.logLevel,configDir:this.config.cwd,engineProtocol:this.engineProtocol},n=>{r.deref()?.logger(n)}),ca++}catch(r){let n=r,i=this.parseInitError(n.message);throw typeof i=="string"?n:new Q(i.message,this.config.clientVersion,i.error_code)}}}logger(r){let n=this.parseEngineResponse(r);if(!!n){if("span"in n){this.config.tracingHelper.createEngineSpan(n);return}n.level=n?.level.toLowerCase()??"unknown",Bp(n)?this.logEmitter.emit("query",{timestamp:new Date,query:n.query,params:n.params,duration:Number(n.duration_ms),target:n.module_path}):Vp(n)?this.loggerRustPanic=new be(this.getErrorMessageWithLink(`${n.message}: ${n.reason} in ${n.file}:${n.line}:${n.column}`),this.config.clientVersion):this.logEmitter.emit(n.level,{timestamp:new Date,message:n.message,target:n.module_path})}}getErrorMessageWithLink(r){return sa({platform:this.platform,title:r,version:this.config.clientVersion,engineVersion:this.versionInfo?.commit,database:this.config.activeProvider,query:this.lastQuery})}parseInitError(r){try{return JSON.parse(r)}catch{}return r}parseRequestError(r){try{return JSON.parse(r)}catch{}return r}on(r,n){r==="beforeExit"?this.beforeExitListener=n:this.logEmitter.on(r,n)}async start(){if(await this.libraryInstantiationPromise,await this.libraryStoppingPromise,this.libraryStartingPromise)return qe(`library already starting, this.libraryStarted: ${this.libraryStarted}`),this.libraryStartingPromise;if(this.libraryStarted)return;let r=async()=>{qe("library starting");try{let n={traceparent:this.config.tracingHelper.getTraceParent()};await this.engine?.connect(JSON.stringify(n)),this.libraryStarted=!0,qe("library started")}catch(n){let i=this.parseInitError(n.message);throw typeof i=="string"?n:new Q(i.message,this.config.clientVersion,i.error_code)}finally{this.libraryStartingPromise=void 0}};return this.libraryStartingPromise=this.config.tracingHelper.runInChildSpan("connect",r),this.libraryStartingPromise}async stop(){if(await this.libraryStartingPromise,await this.executingQueryPromise,this.libraryStoppingPromise)return qe("library is already stopping"),this.libraryStoppingPromise;if(!this.libraryStarted)return;let r=async()=>{await new Promise(i=>setTimeout(i,5)),qe("library stopping");let n={traceparent:this.config.tracingHelper.getTraceParent()};await this.engine?.disconnect(JSON.stringify(n)),this.libraryStarted=!1,this.libraryStoppingPromise=void 0,qe("library stopped")};return this.libraryStoppingPromise=this.config.tracingHelper.runInChildSpan("disconnect",r),this.libraryStoppingPromise}async getDmmf(){await this.start();let r=this.config.tracingHelper.getTraceParent(),n=await this.engine.dmmf(JSON.stringify({traceparent:r}));return this.config.tracingHelper.runInChildSpan({name:"parseDmmf",internal:!0},()=>JSON.parse(n))}version(){return this.versionInfo=this.library?.version(),this.versionInfo?.version??"unknown"}debugPanic(r){return this.library?.debugPanic(r)}async request(r,{traceparent:n,interactiveTransaction:i}){qe(`sending request, this.libraryStarted: ${this.libraryStarted}`);let o=JSON.stringify({traceparent:n}),s=JSON.stringify(r);try{await this.start(),this.executingQueryPromise=this.engine?.query(s,o,i?.id),this.lastQuery=s;let a=this.parseEngineResponse(await this.executingQueryPromise);if(a.errors)throw a.errors.length===1?this.buildQueryError(a.errors[0]):new oe(JSON.stringify(a.errors),{clientVersion:this.config.clientVersion});if(this.loggerRustPanic)throw this.loggerRustPanic;return{data:a,elapsed:0}}catch(a){if(a instanceof Q)throw a;if(a.code==="GenericFailure"&&a.message?.startsWith("PANIC:"))throw new be(this.getErrorMessageWithLink(a.message),this.config.clientVersion);let l=this.parseRequestError(a.message);throw typeof l=="string"?a:new oe(`${l.message}

2023-08-15T15:44:46.355 app[148ed6e1f7d248] mia [info] ^

2023-08-15T15:44:46.368 app[148ed6e1f7d248] mia [info] PrismaClientInitializationError: Error opening a TLS connection: unexpected EOF

2023-08-15T15:44:46.368 app[148ed6e1f7d248] mia [info] at r (/triggerdotdev/node_modules/.pnpm/@prisma+client@4.16.2_prisma@4.16.0/node_modules/@prisma/client/runtime/library.js:108:2574)

2023-08-15T15:44:46.481 app[148ed6e1f7d248] mia [info]  ELIFECYCLE  Command failed with exit code 1.

2023-08-15T15:44:46.762 app[148ed6e1f7d248] mia [info] INFO Main child exited normally with code: 1

2023-08-15T15:44:46.763 app[148ed6e1f7d248] mia [info] INFO Starting clean up.

2023-08-15T15:44:46.764 app[148ed6e1f7d248] mia [info] WARN hallpass exited, pid: 256, status: signal: 15 (SIGTERM)

2023-08-15T15:44:46.771 app[148ed6e1f7d248] mia [info] 2023/08/15 15:44:46 listening on [fdaa:2:4fd7:a7b:184:f397:2946:2]:22 (DNS: [fdaa::3]:53)

2023-08-15T15:44:47.765 app[148ed6e1f7d248] mia [info] [ 11.304070] reboot: Restarting system

2023-08-15T15:44:47.960 runner[148ed6e1f7d248] mia [info] machine did not have a restart policy, defaulting to restart

2023-08-15T15:44:51.763 app[148ed6e1f7d248] mia [info] [ 0.040196] PCI: Fatal: No config space access function found

2023-08-15T15:44:51.988 app[148ed6e1f7d248] mia [info] INFO Starting init (commit: b437b5b)...

2023-08-15T15:44:52.008 app[148ed6e1f7d248] mia [info] INFO Preparing to run: `docker-entrypoint.sh ./scripts/entrypoint.sh` as node

2023-08-15T15:44:52.015 app[148ed6e1f7d248] mia [info] INFO [fly api proxy] listening at /.fly/api

2023-08-15T15:44:52.022 app[148ed6e1f7d248] mia [info] 2023/08/15 15:44:52 listening on [fdaa:2:4fd7:a7b:184:f397:2946:2]:22 (DNS: [fdaa::3]:53)

2023-08-15T15:44:52.025 app[148ed6e1f7d248] mia [info] + [ -n ]

2023-08-15T15:44:52.025 app[148ed6e1f7d248] mia [info] + pnpm --filter @trigger.dev/database db:migrate:deploy

2023-08-15T15:44:53.432 app[148ed6e1f7d248] mia [info] > @trigger.dev/database@0.0.0 db:migrate:deploy /triggerdotdev/packages/database

2023-08-15T15:44:53.432 app[148ed6e1f7d248] mia [info] > prisma migrate deploy

2023-08-15T15:44:54.060 app[148ed6e1f7d248] mia [info] Prisma schema loaded from prisma/schema.prisma

2023-08-15T15:44:54.093 app[148ed6e1f7d248] mia [info] Datasource "db": PostgreSQL database "postgres", schema "public" at "wallofreviews-trigger-v2-fly-demo-db.flycast:5432"

2023-08-15T15:44:54.334 app[148ed6e1f7d248] mia [info] Error: P1011: Error opening a TLS connection: unexpected EOF

2023-08-15T15:44:54.352 app[148ed6e1f7d248] mia [info] undefined

2023-08-15T15:44:54.353 app[148ed6e1f7d248] mia [info] /triggerdotdev/packages/database:

2023-08-15T15:44:54.353 app[148ed6e1f7d248] mia [info]  ERR_PNPM_RECURSIVE_RUN_FIRST_FAIL  @trigger.dev/database@0.0.0 db:migrate:deploy: `prisma migrate deploy`

2023-08-15T15:44:54.353 app[148ed6e1f7d248] mia [info] Exit status 1

2023-08-15T15:44:54.365 app[148ed6e1f7d248] mia [info] + cp packages/database/prisma/schema.prisma apps/webapp/prisma/

2023-08-15T15:44:54.370 app[148ed6e1f7d248] mia [info] + cp node_modules/@prisma/engines/libquery_engine-debian-openssl-1.1.x.so.node apps/webapp/prisma/

2023-08-15T15:44:54.387 app[148ed6e1f7d248] mia [info] + pnpm --filter webapp db:seed

2023-08-15T15:44:54.899 app[148ed6e1f7d248] mia [info] > webapp@1.0.0 db:seed /triggerdotdev/apps/webapp

2023-08-15T15:44:54.899 app[148ed6e1f7d248] mia [info] > node prisma/seed.js

2023-08-15T15:44:54.977 app[148ed6e1f7d248] mia [info] 🔌 setting up prisma client to postgres://postgres@wallofreviews-trigger-v2-fly-demo-db.flycast:5432

2023-08-15T15:44:54.982 app[148ed6e1f7d248] mia [info] 🔌 prisma client connected

2023-08-15T15:44:54.993 app[148ed6e1f7d248] mia [info] prisma:info Starting a postgresql pool with 3 connections.

2023-08-15T15:44:55.177 app[148ed6e1f7d248] mia [info] prisma:error

2023-08-15T15:44:55.177 app[148ed6e1f7d248] mia [info] Invalid `prisma.integrationDefinition.upsert()` invocation:

2023-08-15T15:44:55.177 app[148ed6e1f7d248] mia [info] Error opening a TLS connection: unexpected EOF

2023-08-15T15:44:55.180 app[148ed6e1f7d248] mia [info] PrismaClientInitializationError:

2023-08-15T15:44:55.180 app[148ed6e1f7d248] mia [info] Invalid `prisma.integrationDefinition.upsert()` invocation:

2023-08-15T15:44:55.180 app[148ed6e1f7d248] mia [info] Error opening a TLS connection: unexpected EOF

2023-08-15T15:44:55.180 app[148ed6e1f7d248] mia [info] at Rn.handleRequestError (/triggerdotdev/apps/webapp/prisma/seed.js:174:7589)

2023-08-15T15:44:55.180 app[148ed6e1f7d248] mia [info] at Rn.handleAndLogRequestError (/triggerdotdev/apps/webapp/prisma/seed.js:174:6742)

2023-08-15T15:44:55.180 app[148ed6e1f7d248] mia [info] at /triggerdotdev/apps/webapp/prisma/seed.js:177:3006

2023-08-15T15:44:55.180 app[148ed6e1f7d248] mia [info] at async /triggerdotdev/apps/webapp/prisma/seed.js:177:3221

2023-08-15T15:44:55.180 app[148ed6e1f7d248] mia [info] at async t._executeRequest (/triggerdotdev/apps/webapp/prisma/seed.js:177:11390)

2023-08-15T15:44:55.180 app[148ed6e1f7d248] mia [info] at async fg (/triggerdotdev/apps/webapp/prisma/seed.js:519:60716)

2023-08-15T15:44:55.180 app[148ed6e1f7d248] mia [info] at async pg (/triggerdotdev/apps/webapp/prisma/seed.js:519:61621) {

2023-08-15T15:44:55.180 app[148ed6e1f7d248] mia [info] clientVersion: '4.16.0',

2023-08-15T15:44:55.180 app[148ed6e1f7d248] mia [info] errorCode: undefined

2023-08-15T15:44:55.180 app[148ed6e1f7d248] mia [info] }

2023-08-15T15:44:55.187 app[148ed6e1f7d248] mia [info] undefined

2023-08-15T15:44:55.187 app[148ed6e1f7d248] mia [info] /triggerdotdev/apps/webapp:

2023-08-15T15:44:55.187 app[148ed6e1f7d248] mia [info]  ERR_PNPM_RECURSIVE_RUN_FIRST_FAIL  webapp@1.0.0 db:seed: `node prisma/seed.js`

2023-08-15T15:44:55.187 app[148ed6e1f7d248] mia [info] Exit status 1

2023-08-15T15:44:55.211 app[148ed6e1f7d248] mia [info] + cd /triggerdotdev/apps/webapp

2023-08-15T15:44:55.211 app[148ed6e1f7d248] mia [info] + exec dumb-init pnpm run start

2023-08-15T15:44:55.616 app[148ed6e1f7d248] mia [info] > webapp@1.0.0 start /triggerdotdev/apps/webapp

2023-08-15T15:44:55.616 app[148ed6e1f7d248] mia [info] > cross-env NODE_ENV=production node --max-old-space-size=8192 ./build/server.js

2023-08-15T15:44:56.877 app[148ed6e1f7d248] mia [info] 🔌 setting up prisma client to postgres://postgres@wallofreviews-trigger-v2-fly-demo-db.flycast:5432

2023-08-15T15:44:56.881 app[148ed6e1f7d248] mia [info] 🔌 prisma client connected

2023-08-15T15:44:57.122 app[148ed6e1f7d248] mia [info] {"runnerOptions":{"connectionString":"postgres://postgres:SGbxbMIWU8vqBjh@wallofreviews-trigger-v2-fly-demo-db.flycast:5432","concurrency":5,"pollInterval":1000,"noPreparedStatements":false},"timestamp":"2023-08-15T15:44:57.122Z","name":"webapp","message":"Initializing worker queue with options"}

2023-08-15T15:44:57.573 app[148ed6e1f7d248] mia [info] No PostHog API key, so analytics won't track

2023-08-15T15:44:58.686 app[148ed6e1f7d248] mia [info] ✅ app ready: http://localhost:3000

2023-08-15T15:44:59.788 app[148ed6e1f7d248] mia [info] prisma:info Starting a postgresql pool with 3 connections.

2023-08-15T15:45:00.143 app[148ed6e1f7d248] mia [info] /triggerdotdev/node_modules/.pnpm/@prisma+client@4.16.2_prisma@4.16.0/node_modules/@prisma/client/runtime/library.js:108

2023-08-15T15:45:00.143 app[148ed6e1f7d248] mia [info] You may have to run ${S("prisma generate")} for your changes to take effect.`,this.config.clientVersion);return r}parseEngineResponse(r){if(!r)throw new oe("Response from the Engine was empty",{clientVersion:this.config.clientVersion});try{return JSON.parse(r)}catch{throw new oe("Unable to JSON.parse response from engine",{clientVersion:this.config.clientVersion})}}convertDatasources(r){let n=Object.create(null);for(let{name:i,url:o}of r)n[i]=o;return n}async loadEngine(){if(!this.engine){this.QueryEngineConstructor||(this.library=await this.libraryLoader.loadLibrary(),this.QueryEngineConstructor=this.library.QueryEngine);try{let r=new WeakRef(this);this.engine=new this.QueryEngineConstructor({datamodel:this.datamodel,env:process.env,logQueries:this.config.logQueries??!1,ignoreEnvVarErrors:!0,datasourceOverrides:this.datasourceOverrides,logLevel:this.logLevel,configDir:this.config.cwd,engineProtocol:this.engineProtocol},n=>{r.deref()?.logger(n)}),ca++}catch(r){let n=r,i=this.parseInitError(n.message);throw typeof i=="string"?n:new Q(i.message,this.config.clientVersion,i.error_code)}}}logger(r){let n=this.parseEngineResponse(r);if(!!n){if("span"in n){this.config.tracingHelper.createEngineSpan(n);return}n.level=n?.level.toLowerCase()??"unknown",Bp(n)?this.logEmitter.emit("query",{timestamp:new Date,query:n.query,params:n.params,duration:Number(n.duration_ms),target:n.module_path}):Vp(n)?this.loggerRustPanic=new be(this.getErrorMessageWithLink(`${n.message}: ${n.reason} in ${n.file}:${n.line}:${n.column}`),this.config.clientVersion):this.logEmitter.emit(n.level,{timestamp:new Date,message:n.message,target:n.module_path})}}getErrorMessageWithLink(r){return sa({platform:this.platform,title:r,version:this.config.clientVersion,engineVersion:this.versionInfo?.commit,database:this.config.activeProvider,query:this.lastQuery})}parseInitError(r){try{return JSON.parse(r)}catch{}return r}parseRequestError(r){try{return JSON.parse(r)}catch{}return r}on(r,n){r==="beforeExit"?this.beforeExitListener=n:this.logEmitter.on(r,n)}async start(){if(await this.libraryInstantiationPromise,await this.libraryStoppingPromise,this.libraryStartingPromise)return qe(`library already starting, this.libraryStarted: ${this.libraryStarted}`),this.libraryStartingPromise;if(this.libraryStarted)return;let r=async()=>{qe("library starting");try{let n={traceparent:this.config.tracingHelper.getTraceParent()};await this.engine?.connect(JSON.stringify(n)),this.libraryStarted=!0,qe("library started")}catch(n){let i=this.parseInitError(n.message);throw typeof i=="string"?n:new Q(i.message,this.config.clientVersion,i.error_code)}finally{this.libraryStartingPromise=void 0}};return this.libraryStartingPromise=this.config.tracingHelper.runInChildSpan("connect",r),this.libraryStartingPromise}async stop(){if(await this.libraryStartingPromise,await this.executingQueryPromise,this.libraryStoppingPromise)return qe("library is already stopping"),this.libraryStoppingPromise;if(!this.libraryStarted)return;let r=async()=>{await new Promise(i=>setTimeout(i,5)),qe("library stopping");let n={traceparent:this.config.tracingHelper.getTraceParent()};await this.engine?.disconnect(JSON.stringify(n)),this.libraryStarted=!1,this.libraryStoppingPromise=void 0,qe("library stopped")};return this.libraryStoppingPromise=this.config.tracingHelper.runInChildSpan("disconnect",r),this.libraryStoppingPromise}async getDmmf(){await this.start();let r=this.config.tracingHelper.getTraceParent(),n=await this.engine.dmmf(JSON.stringify({traceparent:r}));return this.config.tracingHelper.runInChildSpan({name:"parseDmmf",internal:!0},()=>JSON.parse(n))}version(){return this.versionInfo=this.library?.version(),this.versionInfo?.version??"unknown"}debugPanic(r){return this.library?.debugPanic(r)}async request(r,{traceparent:n,interactiveTransaction:i}){qe(`sending request, this.libraryStarted: ${this.libraryStarted}`);let o=JSON.stringify({traceparent:n}),s=JSON.stringify(r);try{await this.start(),this.executingQueryPromise=this.engine?.query(s,o,i?.id),this.lastQuery=s;let a=this.parseEngineResponse(await this.executingQueryPromise);if(a.errors)throw a.errors.length===1?this.buildQueryError(a.errors[0]):new oe(JSON.stringify(a.errors),{clientVersion:this.config.clientVersion});if(this.loggerRustPanic)throw this.loggerRustPanic;return{data:a,elapsed:0}}catch(a){if(a instanceof Q)throw a;if(a.code==="GenericFailure"&&a.message?.startsWith("PANIC:"))throw new be(this.getErrorMessageWithLink(a.message),this.config.clientVersion);let l=this.parseRequestError(a.message);throw typeof l=="string"?a:new oe(`${l.message}

2023-08-15T15:45:00.143 app[148ed6e1f7d248] mia [info] ^

2023-08-15T15:45:00.154 app[148ed6e1f7d248] mia [info] PrismaClientInitializationError: Error opening a TLS connection: unexpected EOF

2023-08-15T15:45:00.154 app[148ed6e1f7d248] mia [info] at r (/triggerdotdev/node_modules/.pnpm/@prisma+client@4.16.2_prisma@4.16.0/node_modules/@prisma/client/runtime/library.js:108:2574)

2023-08-15T15:45:00.258 app[148ed6e1f7d248] mia [info]  ELIFECYCLE  Command failed with exit code 1.

2023-08-15T15:45:01.033 app[148ed6e1f7d248] mia [info] INFO Main child exited normally with code: 1

2023-08-15T15:45:01.033 app[148ed6e1f7d248] mia [info] INFO Starting clean up.

2023-08-15T15:45:01.034 app[148ed6e1f7d248] mia [info] WARN hallpass exited, pid: 256, status: signal: 15 (SIGTERM)

2023-08-15T15:45:01.041 app[148ed6e1f7d248] mia [info] 2023/08/15 15:45:01 listening on [fdaa:2:4fd7:a7b:184:f397:2946:2]:22 (DNS: [fdaa::3]:53)

2023-08-15T15:45:02.034 app[148ed6e1f7d248] mia [info] [ 10.307405] reboot: Restarting system

2023-08-15T15:45:02.243 runner[148ed6e1f7d248] mia [info] machine did not have a restart policy, defaulting to restart

2023-08-15T15:45:24.474 app[148ed6e1f7d248] mia [info] [ 0.040617] PCI: Fatal: No config space access function found

2023-08-15T15:45:24.698 app[148ed6e1f7d248] mia [info] INFO Starting init (commit: b437b5b)...

2023-08-15T15:45:24.717 app[148ed6e1f7d248] mia [info] INFO Preparing to run: `docker-entrypoint.sh ./scripts/entrypoint.sh` as node

2023-08-15T15:45:24.725 app[148ed6e1f7d248] mia [info] INFO [fly api proxy] listening at /.fly/api

2023-08-15T15:45:24.732 app[148ed6e1f7d248] mia [info] 2023/08/15 15:45:24 listening on [fdaa:2:4fd7:a7b:184:f397:2946:2]:22 (DNS: [fdaa::3]:53)

2023-08-15T15:45:24.735 app[148ed6e1f7d248] mia [info] + [ -n ]

2023-08-15T15:45:24.735 app[148ed6e1f7d248] mia [info] + pnpm --filter @trigger.dev/database db:migrate:deploy

2023-08-15T15:45:26.501 app[148ed6e1f7d248] mia [info] > @trigger.dev/database@0.0.0 db:migrate:deploy /triggerdotdev/packages/database

2023-08-15T15:45:26.501 app[148ed6e1f7d248] mia [info] > prisma migrate deploy

2023-08-15T15:45:27.154 app[148ed6e1f7d248] mia [info] Prisma schema loaded from prisma/schema.prisma

2023-08-15T15:45:27.184 app[148ed6e1f7d248] mia [info] Datasource "db": PostgreSQL database "postgres", schema "public" at "wallofreviews-trigger-v2-fly-demo-db.flycast:5432"

2023-08-15T15:45:27.444 app[148ed6e1f7d248] mia [info] Error: P1011: Error opening a TLS connection: unexpected EOF

2023-08-15T15:45:27.463 app[148ed6e1f7d248] mia [info] undefined

2023-08-15T15:45:27.464 app[148ed6e1f7d248] mia [info] /triggerdotdev/packages/database:

2023-08-15T15:45:27.464 app[148ed6e1f7d248] mia [info]  ERR_PNPM_RECURSIVE_RUN_FIRST_FAIL  @trigger.dev/database@0.0.0 db:migrate:deploy: `prisma migrate deploy`

2023-08-15T15:45:27.464 app[148ed6e1f7d248] mia [info] Exit status 1

2023-08-15T15:45:27.476 app[148ed6e1f7d248] mia [info] + cp packages/database/prisma/schema.prisma apps/webapp/prisma/

2023-08-15T15:45:27.481 app[148ed6e1f7d248] mia [info] + cp node_modules/@prisma/engines/libquery_engine-debian-openssl-1.1.x.so.node apps/webapp/prisma/

2023-08-15T15:45:27.497 app[148ed6e1f7d248] mia [info] + pnpm --filter webapp db:seed

2023-08-15T15:45:28.012 app[148ed6e1f7d248] mia [info] > webapp@1.0.0 db:seed /triggerdotdev/apps/webapp

2023-08-15T15:45:28.012 app[148ed6e1f7d248] mia [info] > node prisma/seed.js

2023-08-15T15:45:28.090 app[148ed6e1f7d248] mia [info] 🔌 setting up prisma client to postgres://postgres@wallofreviews-trigger-v2-fly-demo-db.flycast:5432

2023-08-15T15:45:28.094 app[148ed6e1f7d248] mia [info] 🔌 prisma client connected

2023-08-15T15:45:28.106 app[148ed6e1f7d248] mia [info] prisma:info Starting a postgresql pool with 3 connections.

2023-08-15T15:45:28.291 app[148ed6e1f7d248] mia [info] prisma:error

2023-08-15T15:45:28.291 app[148ed6e1f7d248] mia [info] Invalid `prisma.integrationDefinition.upsert()` invocation:

2023-08-15T15:45:28.291 app[148ed6e1f7d248] mia [info] Error opening a TLS connection: unexpected EOF

2023-08-15T15:45:28.294 app[148ed6e1f7d248] mia [info] PrismaClientInitializationError:

2023-08-15T15:45:28.294 app[148ed6e1f7d248] mia [info] Invalid `prisma.integrationDefinition.upsert()` invocation:

2023-08-15T15:45:28.294 app[148ed6e1f7d248] mia [info] Error opening a TLS connection: unexpected EOF

2023-08-15T15:45:28.294 app[148ed6e1f7d248] mia [info] at Rn.handleRequestError (/triggerdotdev/apps/webapp/prisma/seed.js:174:7589)

2023-08-15T15:45:28.294 app[148ed6e1f7d248] mia [info] at Rn.handleAndLogRequestError (/triggerdotdev/apps/webapp/prisma/seed.js:174:6742)

2023-08-15T15:45:28.294 app[148ed6e1f7d248] mia [info] at /triggerdotdev/apps/webapp/prisma/seed.js:177:3006

2023-08-15T15:45:28.294 app[148ed6e1f7d248] mia [info] at async /triggerdotdev/apps/webapp/prisma/seed.js:177:3221

2023-08-15T15:45:28.294 app[148ed6e1f7d248] mia [info] at async t._executeRequest (/triggerdotdev/apps/webapp/prisma/seed.js:177:11390)

2023-08-15T15:45:28.294 app[148ed6e1f7d248] mia [info] at async fg (/triggerdotdev/apps/webapp/prisma/seed.js:519:60716)

2023-08-15T15:45:28.294 app[148ed6e1f7d248] mia [info] at async pg (/triggerdotdev/apps/webapp/prisma/seed.js:519:61621) {

2023-08-15T15:45:28.294 app[148ed6e1f7d248] mia [info] clientVersion: '4.16.0',

2023-08-15T15:45:28.294 app[148ed6e1f7d248] mia [info] errorCode: undefined

2023-08-15T15:45:28.294 app[148ed6e1f7d248] mia [info] }

2023-08-15T15:45:28.301 app[148ed6e1f7d248] mia [info] undefined

2023-08-15T15:45:28.302 app[148ed6e1f7d248] mia [info] /triggerdotdev/apps/webapp:

2023-08-15T15:45:28.302 app[148ed6e1f7d248] mia [info]  ERR_PNPM_RECURSIVE_RUN_FIRST_FAIL  webapp@1.0.0 db:seed: `node prisma/seed.js`

2023-08-15T15:45:28.302 app[148ed6e1f7d248] mia [info] Exit status 1

2023-08-15T15:45:28.324 app[148ed6e1f7d248] mia [info] + cd /triggerdotdev/apps/webapp

2023-08-15T15:45:28.324 app[148ed6e1f7d248] mia [info] + exec dumb-init pnpm run start

2023-08-15T15:45:28.731 app[148ed6e1f7d248] mia [info] > webapp@1.0.0 start /triggerdotdev/apps/webapp

2023-08-15T15:45:28.731 app[148ed6e1f7d248] mia [info] > cross-env NODE_ENV=production node --max-old-space-size=8192 ./build/server.js

2023-08-15T15:45:30.057 app[148ed6e1f7d248] mia [info] 🔌 setting up prisma client to postgres://postgres@wallofreviews-trigger-v2-fly-demo-db.flycast:5432

2023-08-15T15:45:30.064 app[148ed6e1f7d248] mia [info] 🔌 prisma client connected

2023-08-15T15:45:30.295 app[148ed6e1f7d248] mia [info] {"runnerOptions":{"connectionString":"postgres://postgres:SGbxbMIWU8vqBjh@wallofreviews-trigger-v2-fly-demo-db.flycast:5432","concurrency":5,"pollInterval":1000,"noPreparedStatements":false},"timestamp":"2023-08-15T15:45:30.295Z","name":"webapp","message":"Initializing worker queue with options"}

2023-08-15T15:45:30.730 app[148ed6e1f7d248] mia [info] No PostHog API key, so analytics won't track

2023-08-15T15:45:31.886 app[148ed6e1f7d248] mia [info] ✅ app ready: http://localhost:3000

2023-08-15T15:45:32.558 app[148ed6e1f7d248] mia [info] prisma:info Starting a postgresql pool with 3 connections.

2023-08-15T15:45:33.495 app[148ed6e1f7d248] mia [info] /triggerdotdev/node_modules/.pnpm/@prisma+client@4.16.2_prisma@4.16.0/node_modules/@prisma/client/runtime/library.js:108

2023-08-15T15:45:33.495 app[148ed6e1f7d248] mia [info] You may have to run ${S("prisma generate")} for your changes to take effect.`,this.config.clientVersion);return r}parseEngineResponse(r){if(!r)throw new oe("Response from the Engine was empty",{clientVersion:this.config.clientVersion});try{return JSON.parse(r)}catch{throw new oe("Unable to JSON.parse response from engine",{clientVersion:this.config.clientVersion})}}convertDatasources(r){let n=Object.create(null);for(let{name:i,url:o}of r)n[i]=o;return n}async loadEngine(){if(!this.engine){this.QueryEngineConstructor||(this.library=await this.libraryLoader.loadLibrary(),this.QueryEngineConstructor=this.library.QueryEngine);try{let r=new WeakRef(this);this.engine=new this.QueryEngineConstructor({datamodel:this.datamodel,env:process.env,logQueries:this.config.logQueries??!1,ignoreEnvVarErrors:!0,datasourceOverrides:this.datasourceOverrides,logLevel:this.logLevel,configDir:this.config.cwd,engineProtocol:this.engineProtocol},n=>{r.deref()?.logger(n)}),ca++}catch(r){let n=r,i=this.parseInitError(n.message);throw typeof i=="string"?n:new Q(i.message,this.config.clientVersion,i.error_code)}}}logger(r){let n=this.parseEngineResponse(r);if(!!n){if("span"in n){this.config.tracingHelper.createEngineSpan(n);return}n.level=n?.level.toLowerCase()??"unknown",Bp(n)?this.logEmitter.emit("query",{timestamp:new Date,query:n.query,params:n.params,duration:Number(n.duration_ms),target:n.module_path}):Vp(n)?this.loggerRustPanic=new be(this.getErrorMessageWithLink(`${n.message}: ${n.reason} in ${n.file}:${n.line}:${n.column}`),this.config.clientVersion):this.logEmitter.emit(n.level,{timestamp:new Date,message:n.message,target:n.module_path})}}getErrorMessageWithLink(r){return sa({platform:this.platform,title:r,version:this.config.clientVersion,engineVersion:this.versionInfo?.commit,database:this.config.activeProvider,query:this.lastQuery})}parseInitError(r){try{return JSON.parse(r)}catch{}return r}parseRequestError(r){try{return JSON.parse(r)}catch{}return r}on(r,n){r==="beforeExit"?this.beforeExitListener=n:this.logEmitter.on(r,n)}async start(){if(await this.libraryInstantiationPromise,await this.libraryStoppingPromise,this.libraryStartingPromise)return qe(`library already starting, this.libraryStarted: ${this.libraryStarted}`),this.libraryStartingPromise;if(this.libraryStarted)return;let r=async()=>{qe("library starting");try{let n={traceparent:this.config.tracingHelper.getTraceParent()};await this.engine?.connect(JSON.stringify(n)),this.libraryStarted=!0,qe("library started")}catch(n){let i=this.parseInitError(n.message);throw typeof i=="string"?n:new Q(i.message,this.config.clientVersion,i.error_code)}finally{this.libraryStartingPromise=void 0}};return this.libraryStartingPromise=this.config.tracingHelper.runInChildSpan("connect",r),this.libraryStartingPromise}async stop(){if(await this.libraryStartingPromise,await this.executingQueryPromise,this.libraryStoppingPromise)return qe("library is already stopping"),this.libraryStoppingPromise;if(!this.libraryStarted)return;let r=async()=>{await new Promise(i=>setTimeout(i,5)),qe("library stopping");let n={traceparent:this.config.tracingHelper.getTraceParent()};await this.engine?.disconnect(JSON.stringify(n)),this.libraryStarted=!1,this.libraryStoppingPromise=void 0,qe("library stopped")};return this.libraryStoppingPromise=this.config.tracingHelper.runInChildSpan("disconnect",r),this.libraryStoppingPromise}async getDmmf(){await this.start();let r=this.config.tracingHelper.getTraceParent(),n=await this.engine.dmmf(JSON.stringify({traceparent:r}));return this.config.tracingHelper.runInChildSpan({name:"parseDmmf",internal:!0},()=>JSON.parse(n))}version(){return this.versionInfo=this.library?.version(),this.versionInfo?.version??"unknown"}debugPanic(r){return this.library?.debugPanic(r)}async request(r,{traceparent:n,interactiveTransaction:i}){qe(`sending request, this.libraryStarted: ${this.libraryStarted}`);let o=JSON.stringify({traceparent:n}),s=JSON.stringify(r);try{await this.start(),this.executingQueryPromise=this.engine?.query(s,o,i?.id),this.lastQuery=s;let a=this.parseEngineResponse(await this.executingQueryPromise);if(a.errors)throw a.errors.length===1?this.buildQueryError(a.errors[0]):new oe(JSON.stringify(a.errors),{clientVersion:this.config.clientVersion});if(this.loggerRustPanic)throw this.loggerRustPanic;return{data:a,elapsed:0}}catch(a){if(a instanceof Q)throw a;if(a.code==="GenericFailure"&&a.message?.startsWith("PANIC:"))throw new be(this.getErrorMessageWithLink(a.message),this.config.clientVersion);let l=this.parseRequestError(a.message);throw typeof l=="string"?a:new oe(`${l.message}

2023-08-15T15:45:33.495 app[148ed6e1f7d248] mia [info] ^

2023-08-15T15:45:33.504 app[148ed6e1f7d248] mia [info] PrismaClientInitializationError: Error opening a TLS connection: unexpected EOF

2023-08-15T15:45:33.504 app[148ed6e1f7d248] mia [info] at r (/triggerdotdev/node_modules/.pnpm/@prisma+client@4.16.2_prisma@4.16.0/node_modules/@prisma/client/runtime/library.js:108:2574)

2023-08-15T15:45:33.567 app[148ed6e1f7d248] mia [info]  ELIFECYCLE  Command failed with exit code 1.

2023-08-15T15:45:33.739 app[148ed6e1f7d248] mia [info] INFO Main child exited normally with code: 1

2023-08-15T15:45:33.739 app[148ed6e1f7d248] mia [info] INFO Starting clean up.

2023-08-15T15:45:33.740 app[148ed6e1f7d248] mia [info] WARN hallpass exited, pid: 256, status: signal: 15 (SIGTERM)

2023-08-15T15:45:33.748 app[148ed6e1f7d248] mia [info] 2023/08/15 15:45:33 listening on [fdaa:2:4fd7:a7b:184:f397:2946:2]:22 (DNS: [fdaa::3]:53)

2023-08-15T15:45:34.741 app[148ed6e1f7d248] mia [info] [ 10.303148] reboot: Restarting system

2023-08-15T15:45:34.896 runner[148ed6e1f7d248] mia [info] machine has reached its max restart count (10)

--- a/docs/documentation/guides/self-hosting/flyio.mdx
+++ b/docs/documentation/guides/self-hosting/flyio.mdx
@@ -74,10 +74,11 @@ Both of these secrets should be set to the base URL of your fly application. For
 
 3. `DIRECT_URL`
 
-This needs to be set to the database connection string that was printed to your terminal after the creation step above:
+This needs to match the value of `DATABASE_URL` which was printed to your terminal after the creation step above:
 
 ```sh
-Connection string: postgres://postgres:<PASSWORD>@<fly db name>.flycast:5432
+The following secret was added to <app name>:
+  DATABASE_URL=postgres://postgres:<PASSWORD>@<fly db name>.flycast:5432/<app name>?sslmode=disable
 ```
 
 ### Optional
@@ -111,7 +112,7 @@ fly secrets set \
   SESSION_SECRET=<random string> \
   LOGIN_ORIGIN="https://<fly app name>.fly.dev" \
   APP_ORIGIN="https://<fly app name>.fly.dev" \
-  DIRECT_URL="postgres://postgres:<PASSWORD>@<fly db name>.flycast:5432" \
+  DIRECT_URL="postgres://postgres:<PASSWORD>@<fly db name>.flycast:5432/<app name>?sslmode=disable" \
   FROM_EMAIL="Acme Inc. <hello@yourdomain.com>" \
   REPLY_TO_EMAIL="Acme Inc. <reply@yourdomain.com>" \
   RESEND_API_KEY=<your API Key> \

